src/venue.md

```diff
@@ -17,6 +17,12 @@ The site has accessible toilets, baby changing facilities, a lift to all floors,
 
 Parking isn't generally available, but parking for disabled visitors can be arranged in advance. If this is something you need, please [contact us](https://2025.pyconuk.org/contact/).
 
+### Planning your trip
+
+We've put together a shortlist of [hotels](#hotels) and [hostels](#hostels) in Manchester.
+
+There are also some tips for people travelling by [train](#by-train), [plane](#by-plane), [car](#by-car), or&nbsp;[coach](#by-coach).
```

Comment:
This is probably fine, but I wonder if adding a TOC to the page might be worth considering instead?

I'm not sure if there's a built-in way in jekyll at the mo, but if you think it's worth us trying it, we could use the example on: https://darn.es/adding-heading-links-to-your-jekyll-blog/ ?

If you think it'll not be quite so user friendly then feel free to disregard this!

Reply:
Yeah, makes sense- thanks!